src/js/media-playback-rate-button.js

```diff
@@ -60,12 +63,22 @@ class MediaPlaybackRateButton extends MediaChromeButton {
     super.attributeChangedCallback(attrName, oldValue, newValue);
   }
 
+  get rates() {
```

Comment:
this seems like scope-creep.
This getter should return a DOMTokenList, while the setter should accept an array.

Reply:
Cool I agree let's just drop all of the property changes in this PR. @luwes when we re-add we'll be sure to account for your private variable naming convention prefs.